docs/HowTo/Configure/Genesis-File.md

```diff
@@ -1,20 +1,23 @@
-description: Configuring network using genesis file  
+description: Configuring a network using the genesis file
 <!--- END of page meta data -->
 
-# Creating Hyperledger Besu Genesis File 
+# Creating the Hyperledger Besu genesis file
```

Comment:
Did you search the doc repo for links to this heading as discussed on Friday?

Reply:
Yes, every time I make a heading change I search for links to that heading throughout the repo, and will update any I find and include them in the PR.